openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -771,3 +771,17 @@ a[href].link-text-grey,
     }
   }
 }
+
+// Common add edit form page styling
+.steps-form-container {
+  width: 70%;
+  margin: 16px auto 0;
+  padding-bottom: 16px;
+
+  .ant-form-item {
+    margin: 0px;
+  }
+  .ant-card {
+    border-radius: 10px;
+  }
+}
```

Comment:
Can we avoid adding it to app.less -- Something generic to have
Let's have specific less file to control specific component